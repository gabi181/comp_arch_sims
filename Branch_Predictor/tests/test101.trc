2 4 26 1 global_history local_tables not_using_share
0x13788E50 N 0x7941D358
0x13788E50 T 0x7941D358
0x13788E50 T 0x7941D358
0x13788E50 N 0x7941D358
0x13788E50 T 0x7941D358
0x13788E50 T 0x7941D358
0x13788E50 T 0x7941D358
0x13788E50 T 0x7941D358
0x13788E50 N 0x7941D358
0x13788E50 T 0x7941D358
0x13788E50 T 0x7941D358
