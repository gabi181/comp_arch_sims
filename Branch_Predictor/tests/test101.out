0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 N 0x13788e54
0x13788e50 T 0x7941d358
flush_num: 7, br_num: 11, size: 182b
